src/main/scala/org/hammerlab/guacamole/commands/StructuralVariantCaller.scala

```diff
@@ -180,6 +182,99 @@ object StructuralVariant {
       graph
     }
 
+    case class SVClique(
+      reads: Set[PairedMappedRead],
+      span: GenomeRange,
+      wiggle: Long
+    )
+
+    def findOneClique(g: PairedReadGraph, maxNormalInsertSize: Int): SVClique = {
+      // Seed the clique with the source of the lowest-weight edge.
+      val edges = g.edges.toList.sortBy(_.weight)
+      val bestEdge = edges.head
+
+      val initPair = bestEdge.nodes.minBy(_.minPos).value
+      var (pairMin, svStart, svEnd, _) = initPair.startsAndStops
+
+      // this is the "wiggle room": by how much can the SV shrink and still be compatible with all reads in the clique?
+      var wiggle = initPair.insertSize - maxNormalInsertSize - (svEnd - svStart)
```

Comment:
I'm confused by this calculation. What size is the SV shrinking from? Presumably `svEnd - svStart`? Either way, the read-pair currently spans `initPair.insertSize` and must still be at least `maxNormalInsertSize`, so the amount it (and therefore its gap) can shrink is `initPair.insertSize - maxNormalInsertSize`. Why subtracting `(svEnd - svStart)`?